projects/wallet-template/src/containers/WalletPopup/pages/Accounts/ImportAccounts.tsx

```diff
@@ -0,0 +1,399 @@
+import { ArrowLeft, Key, NotepadText, ChevronDown, Check } from "lucide-react"
+import { rpc } from "../../api"
+import { SubmitHandler, useForm, Controller } from "react-hook-form"
+import { useNavigate } from "react-router-dom"
+import {
+  entropyToMiniSecret,
+  sr25519,
+  ed25519,
+  ecdsa,
+  mnemonicToEntropy,
+} from "@polkadot-labs/hdkd-helpers"
+import { fromHex, toHex } from "@polkadot-api/utils"
+import { bytesToHex } from "@noble/ciphers/utils"
+import {
+  CreateDeriveFn,
+  ecdsaCreateDerive,
+  ed25519CreateDerive,
+  sr25519CreateDerive,
+} from "@polkadot-labs/hdkd"
+import { networks } from "./networks"
+
+const createDeriveFnMap: Record<Scheme, CreateDeriveFn> = {
+  Sr25519: sr25519CreateDerive,
+  Ed25519: ed25519CreateDerive,
+  Ecdsa: ecdsaCreateDerive,
+}
+
+type Scheme = "Sr25519" | "Ed25519" | "Ecdsa"
+
+type Tab =
+  | {
+      _type: "mnemonic"
```

Comment:
`type` is reserved in react hook form

Reply:
However, react-hook-form should not dictate the API for an extension RPC endpoint.
Therefore, there could be a TS type for a react-hook-form and another type for the RPC endpoint.
Then, before submitting the form fields the `_type` could be transformed to `type`